src/types/ErrorType.ts

```diff
@@ -0,0 +1,8 @@
+export enum ErrorType {
+  default = '',
+  load = 'Unable to load todos',
+  title = 'Title should not be empty',
+  add = 'Unable to add a todo',
+  delete = 'Unable to delete a todo',
+  update = 'Unable to update a todo',
```

Comment:
Consider using uppercase for enum members to follow common conventions, enhancing readability and clarity.